srcpkgs/st/template

```diff
@@ -29,6 +29,7 @@ post_install() {
 
 st-terminfo_package() {
 	short_desc+=" - terminfo data"
+	depends="ncurses"
 	pkg_install() {
```

Comment:
Can you document in the man page the purpose of this change? It's rather unclear to me.

Reply:
In what manpage?

Reply:
Don't review PRs while sleepy >.<

I meant the commit message.

Reply:
You are allowed to use more than 1 line

- 1 line
- blank line
- more text

Reply:
Done.